### PR TITLE
Set better error messages when installing skupper declaratively

### LINKED ISSUE #1352: Declarative Setup - Poor Skupper Status Error Message
**Describe the bug**
Following the example here: https://github.com/skupperproject/skupper-example-declarative

The first time I ran skupper status this is what I saw.
```
$ skupper status
Error: configmaps "skupper-network-status" not found
```

I get why this is happening, but it's not a particularly friendly error message for a user.

**How To Reproduce**
Be fast.

**Expected behavior**
Maybe prefix it with something like "Status unavailable" or "Status pending"?

**Environment details**
 - Skupper CLI: 1.5.2
 - Platform: openshift, k8s

### PR DESCRIPTION
**Pull request by nluaces**
Fixes #1352

Reviewed commands after installing skupper declaratively:
- skupper service status
- skupper link status
- skupper status
- skupper network status

### CODE REVIEW

**Review by grs — APPROVED** [review]

File: cmd/skupper/skupper_kube_site.go [hunk 1]
```diff
@@ -246,6 +246,9 @@ func (s *SkupperKubeSite) Status(cmd *cobra.Command, args []string) error {
 	if errStatus != nil && strings.HasPrefix(errStatus.Error(), "Skupper is not installed") {
 		fmt.Printf("Skupper is not enabled in namespace '%s'", cli.GetNamespace())
```
[the remark this answers is not in the document]
Comment (nluaces) [reply cmd/skupper/skupper_kube_site.go:247]: Sure! thanks!

File: cmd/skupper/skupper_kube_site.go [hunk 2]
```diff
@@ -246,6 +246,9 @@ func (s *SkupperKubeSite) Status(cmd *cobra.Command, args []string) error {
...
 		return nil
+	} else if errStatus != nil && strings.Contains(errStatus.Error(), "\"skupper-network-status\" not found") {
+		fmt.Println("Status pending...")
```
[the remark this answers is not in the document]
Comment (nluaces) [reply cmd/skupper/skupper_kube_site.go:250]: Yes, it is. I could only reproduce it with network status command, I will fix it here as well. Thanks!

File: cmd/skupper/skupper_kube_site.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (nluaces) [reply cmd/skupper/skupper_kube_site.go]: fixed in fbad1115bcc7fbb13e7f832a189d1a82213ea742

File: cmd/skupper/skupper_kube_site.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (nluaces) [reply cmd/skupper/skupper_kube_site.go:250]: fixed in 6bea63746091fc010a3d35edb2822b87c06cedd3

**Review by fgiorgetti — APPROVED** [review]

**Review by c-kruse — APPROVED** [review]

**Pull request merged**